src/controllers/user-management/permission.controller.ts

```diff
@@ -225,11 +225,14 @@ export class PermissionController {
         if (req.user.permissions.isGlobalAdmin) {
             return this.permissionService.getAllPermissions(query);
         } else {
-            const allowedOrganizations = req.user.permissions.getAllOrganizationsWithUserAdmin();
-            return this.permissionService.getAllPermissionsInOrganizations(
-                allowedOrganizations,
-                query
-            );
+            if (query.organisationId === undefined) {
```

Comment:
Minor thing, but I think that I would either had made and if, elseif and else statement instead of the if statement inside of else. Or else I would had said
```
if (req.user.permissions.isGlobalAdmin) {
            return this.permissionService.getAllPermissions(query);
        } else if (query.organisationId === undefined) {
            const allowedOrganizations = req.user.permissions.getAllOrganizationsWithUserAdmin();
            return this.permissionService.getAllPermissionsInOrganizations(
                allowedOrganizations,
                query
            );
        } else {
            return this.permissionService.getAllPermissions(query);
        }
```
But just a thought :-)! This is working fine aswell.

Reply:
No need for an else when there's returns in the if statements

Reply:
Why not saying 
```
      if (req.user.permissions.isGlobalAdmin) {
            return this.permissionService.getAllPermissions(query);
        }
        if (query.organisationId === undefined) {
            const allowedOrganizations = req.user.permissions.getAllOrganizationsWithUserAdmin();
            return this.permissionService.getAllPermissionsInOrganizations(
                allowedOrganizations,
                query
            );
        }
        return this.permissionService.getAllPermissions(query);
```

And getting rid of the nesting then? :-)